rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -1828,9 +1835,30 @@ impl UpgradePath {
             SnsCanisterType::Index => new_latest_version.index_wasm_hash = wasm_hash.to_vec(),
         }
 
+        println!(
+            "Do we get here? Here is the upgrade path: {:?}",
```

Comment:
The comment is unclear to me, is this intended to stay?

Reply:
Was leftover from debugging, thanks